project/frontend/src/components/RecentBuys/RecentBuys.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import classes from "./RecentBuys.module.css";
+import { Slide } from "react-slideshow-image";
+import "react-slideshow-image/dist/styles.css";
+import RecentBuy from "./RecentBuy/RecentBuy";
+
+const RecentBuys = (props) => {
```

Comment:
RecentBuys vs RecentBuy is hard to distinguish at a glance.   May be RecentBuyCollection? Up to you

Reply:
Sure, I will rename to RecentBuysList

Reply:
Done